client-v2/src/actions/Collections.ts

```diff
@@ -102,6 +103,31 @@ function fetchStaleOpenCollections(
   };
 }
 
+// These are collections which exist in the config but have never had any content
+// added to them so they will not have their own collection file. They may be newly
+// created collections or fully automated collections.
+function getCollectionActionForMissingCollection(
```

Comment:
Thought a separate function is tidier than a lot of if/elses in the old getCollectionActions function